packages/dev/core/src/Meshes/thinInstanceMesh.ts

```diff
@@ -168,6 +168,10 @@ Mesh.prototype.thinInstanceSetMatrixAt = function (index: number, matrix: DeepIm
 };
 
 Mesh.prototype.thinInstanceSetAttributeAt = function (kind: string, index: number, value: Array<number>, refresh: boolean = true): boolean {
+    if (kind === VertexBuffer.ColorKind) {

```

Comment:
I think this block of code should be added in all the thin instance methods that are taking a "kind" as a parameter?

Reply:
OK sounds good, to help finish the back compat…  Pretty sure I got all of them - it was already being done for thinInstanceSetBuffer. :)